tests/door43_tools_tests/test_templaters.py

```diff
@@ -48,9 +48,9 @@ def testTemplaterTaComplete(self):
         test_file_path = self.extractZipFiles(test_folder_name)
         success = self.doTemplater('ta', test_file_path)
         self.verifyTaTemplater(success, expect_success, self.out_dir,
-                               ['checking.html', 'intro.html', 'process.html', 'translate.html'])
+                               ['01-intro.html', '02-process.html', '03-translate.html', '04-checking.html'])
```

Comment:
It should matter, as if checking.html is created by the converter from checking.md (not templated), and then when the converted files are templated, there should be a 01-checking.html in the out_dir. So not sure what you mean by this not testing anything.